cities_light/downloader.py

```diff
@@ -0,0 +1,54 @@
+"""Data downloader."""
+
+from __future__ import unicode_literals
+
+import time
+import os
+
+try:
+    from urllib.request import urlopen
+    from urllib.parse import urlparse
+except ImportError:
+    from urllib import urlopen
+    from urlparse import urlparse
+
+
+class Downloader(object):
+
+    """Data downloader."""
+
+    def download(self, source, destination, logger, force=False):
+        """Data download method."""
+        # Prevent copying itself
+        parsed_source = urlparse(source)
+        if parsed_source.scheme == 'file':
```

Comment:
Oops, seems like we duplicated this for testing ...

https://github.com/yourlabs/django-cities-light/blame/stable/3.x.x/cities_light/geonames.py#L31-L34

So maybe, please be careful when you rebase to remove one of the two implementations of local file download in the code !